config/initializers/load_application_yml.rb

```diff
@@ -0,0 +1,5 @@
+# config/initializers/load_application_yml.rb
+config = YAML.load_file(Rails.root.join('config', 'application.yml'))[Rails.env]
```

Comment:
Let us ensure YAML aliases are enabled usually it breaks the code:
```suggestion
config = YAML.safe_load(File.read(Rails.root.join('config', 'application.yml')), aliases: true)[Rails.env]
```

Reply:
This was handled